static/js/util/util.js

```diff
@@ -386,7 +386,7 @@ export function programCourseInfo(program: Program): Object {
   if (program.courses) {
     totalCourses = program.courses.length;
     passedCourses = program.courses.filter(
-      course => course.runs.length > 0 && course.runs[0].status === STATUS_PASSED
+      course => _.some(course.runs, ["status", STATUS_PASSED])
```

Comment:
Yes, I understand that. However, it's not very clear just by looking at the code. That's why I'm asking you to add a comment, as I demonstrated in my comment above.